pkg/linter/lib/src/lint_codes.g.dart

```diff
@@ -623,8 +623,9 @@ class LinterLintCode extends LintCode {
 
   static const LintCode do_not_use_environment = LinterLintCode(
     LintNames.do_not_use_environment,
-    "Invalid use of an environment declaration.",
-    correctionMessage: "Try removing the environment declaration usage.",
+    "Avoid using environment values like '{0}' which create hidden global state.",
+    correctionMessage:
+        "Try using 'Platform.environment' for runtime access or remove environment-dependent code.",
```

Comment:
This sounds misleading. Using `Platform.environment` is runtime access to *something completely different*.
Maybe it's what the user wanted, but it's not a just a switch from compile-time to runtime access.